### ISSUE TITLE
Confirm sort with button

### ISSUE THREAD

**Issue by godhandkiller** [optimistdigital/nova-sortable#30]
Hello, I have a project where the client handles important information, so every changes that he makes it affects a lot of other users. He wants to be able to drag and drop items to sort them but I want if I can make it so the user has to click on a Confirm button before the sort changes on the database.

**Comment by Navi2016** [comment]
Agree, a unexpected sort could cause troubles. People click all over the place so this happens frequently.
